bin/initoverlayfs-install

```diff
@@ -66,22 +116,17 @@ if [ -z "$kver" ]; then
   kver="$(uname -r)"
 fi
 
-initoverlayfs_conf="/etc/initoverlayfs.conf"
-if ! [ -e "$initoverlayfs_conf" ]; then
-  boot_partition=$(cat /etc/fstab | grep "/boot.*ext4" | awk '{print $1}')
-  echo -e "bootfs $boot_partition\nbootfstype ext4\ninitoverlayfs_builder dracut -H -f -v -M --reproducible -o \"initoverlayfs\"\ninitrd_builder dracut -H -f -v -M --reproducible -m \"kernel-modules udev-rules initoverlayfs\" -o \"bash systemd systemd-initrd i18n kernel-modules-extra rootfs-block dracut-systemd usrmount base fs-lib microcode_ctl-fw_dir_override shutdown nss-softokn\"\nudev_trigger udevadm trigger --type=devices --action=add --subsystem-match=module --subsystem-match=block --subsystem-match=virtio --subsystem-match=pci --subsystem-match=nvme\n" > $initoverlayfs_conf
-fi
-
-no_kern=""
```

Comment:
I added this logic for the case where there is no kernel present in this directory, some microVMs and containers are such examples:

```
no_kern=""
if ! compgen -G /boot/vmlinu* > /dev/null; then
  no_kern="--no-kernel"
fi
```

we should re-add.

Reply:
You know what screw it, we can catch in a follow on PR :smile: